common/wuwebview/wuwebview.cpp

```diff
@@ -142,9 +144,21 @@ class WuExpandedResultBuffer : public CInterface, implements IPTreeNotifyEvent
         assertex(!finalized);
         if (!dll)
             return;
+        BoolHash uniqueResultNames;
         Owned<IPropertyTreeIterator> iter = manifest.getElements("Resource[@type='RESULT_XSD']");
         ForEach(*iter)
-            appendSchemaResource(iter->query(), dll);
+        {
+            IPropertyTree& res = iter->query();
+            const char* name = res.queryProp("@name");
+            if (name && *name)
+            {
+                bool* found = uniqueResultNames.getValue(name);
+                if (found && *found)
+                    continue;
+                uniqueResultNames.setValue(name, true);
+            }
+            appendSchemaResource(res, dll);
```

Comment:
I am not sure. Since this method may be used for different reasons, I think it may be safe to keep it. Please let me know if otherwise.